### ISSUE TITLE
Silicon Straits - Senior Backend Developer - HCMC - FT

### ISSUE THREAD

**Issue by kentnguyen** [awesome-jobs/vietnam#215]
![Silicon Straits logo](http://www.siliconstraits.vn/favicon-196x196.png)
## Location
- 147 Tôn Dật Tiên, District 7, Saigon, Vietnam
## Salary Expectation
- $1500+/month
## Requirements

We keep this simple, we are looking for senior/lead level with specific skills:
- Spring MVC / Django / Rails framework.
- Message queue/broker technologies.
- Involved directly with scaling up existing projects.
- Experience with custom production deployment (Heroku doesn't count).
- Can setup CI/CD.
- Micro-services understanding.
- 3+ years Software Engineering experience.
### Must Have

And of course we assume you have all the basics in place:
- Git, version control practices.
- Develop REST APIs.
- Understand IaaS, PaaS, SaaS.
- Linux/Mac tools.
- Be a team player, we don't appreciate lone superhero.
### Responsibilities
- Design & implement back-end code.
- Work directly with business decision makers to define requirements.

Basically you'll code a lot.
### You’ll be responsible for

T3VyIGN1cnJlbnQgcHJvamVjdCB1c2VzIGEgbWl4IG9mIFNwcmluZyBNVkMsIFB5dGhvbiBhbmQgUm9SIGluIHRoZSBiYWNrZW5kIGFuZCBSZWFjdEpTIGluIHRoZSBmcm9udCBlbmQuIFRoZSBwcm9qZWN0IGp1c3QgZ290IHN0YXJ0ZWQuIEl0IGlzIGEgbGFyZ2Ugc3lzdGVtIGJ1aWxkaW5nIGZyb20gc2NyYXRjaCwgZm9yIGEgY29udmVuaWVudCByZXRhaWwgY2hhaW4gdGhhdCB3aWxsIGdvIGxpdmUgc29vbiB3aXRoaW4gb25lIHllYXIuIFdlIGNvdWxkbid0IGRpc2Nsb3NlIG1vcmUgaW5mb3JtYXRpb24sIGJ1dCBpZiB5b3UgdGFrZSB1cCB0aGUgaW50ZXJ2aWV3LCB3ZSB3aWxsIHRlbGwgeW91Lg==

Your work will be legendary.
### Contact

Tell us about yourself via hr@siliconstraits.com and we’ll get back to you right away. Contact us **before 31st May 2016**.

**Comment by joneslee85** [comment]
@kentnguyen thanks for job post. Top notch post btw. It would be nice if you could add SSS logo. Please feel free to close the job post on 31st May. Cheers
